src/App.jsx

```diff
@@ -272,14 +272,15 @@ function App() {
     protect: "unlock",
```

Comment:
Move array of customers to different file.

Reply:
i dont understand well why i need to move to another file

Reply:
To keep your component simple.